stacks-signer/src/v0/signer.rs

```diff
@@ -406,8 +447,13 @@ impl Signer {
     }
 
     /// Handle the block validate response returned from our prior calls to submit a block for validation
-    fn handle_block_validate_response(&mut self, block_validate_response: &BlockValidateResponse) {
-        info!("{self}: Received a block validate response: {block_validate_response:?}");
+    fn handle_block_validate_response(
+        &mut self,
+        stacks_client: &StacksClient,
+        block_validate_response: &BlockValidateResponse,
+    ) {
+        debug!("{self}: Received a block validate response: {block_validate_response:?}");
```

Comment:
I think it's useful to keep this as `info!`. It's helpful to figure out where things go wrong between signer and stacks-node without the need to dig through all of the debug messages.